javascript/operators/import_meta.json

```diff
@@ -68,15 +68,7 @@
                 "version_added": false
               },
               "nodejs": {
```

Comment:
Use an array; don't remove old versions' information.

Reply:
Sounds good. I wasn't sure how to handle 20.0.0 — should I include all three versions?

Reply:
Done!